WME_Image_Overlays.user.js

```diff
@@ -888,26 +859,22 @@ function setTranslations(translations) {
 }
 
 // Create a tab and possibly receive a previous tab to restore (usually in case of a mode change)
-function addTab(recoveredTab) {
-  var userInfo = document.getElementById('user-info'),
-      tabHandles = userInfo.querySelector('.nav-tabs'),
-      tabs = userInfo.querySelector('.tab-content'),
-      tabHandle = document.createElement('li'),
-      tab = document.createElement('div');
-  tabHandle.innerHTML = '<a href="#sidepanel-imageoverlays" data-toggle="tab" title="' + I18n.t('image_overlays.tab_title') + '"><span class="fa fa-picture-o"></span></a>';
-  if (recoveredTab) {
-    tab = recoveredTab;
-  } else {
-    tab.id = 'sidepanel-imageoverlays';
-    tab.className = 'tab-pane';
-    tab.style.position = 'relative';
-  }
-  tabHandles.appendChild(tabHandle);
-  $(tabHandle.childNodes[0]).tooltip();
-  tabs.appendChild(tab);
-  // Make the tab content fill up the height it gets. It is the only variable length box in this column-oriented flex container, so this shouldn't affect anyone
-  tabs.style.flexGrow = '1';
-  return tab;
+async function addTab() {
+  var tab = document.createElement('div');
+  tab.id = 'sidepanel-imageoverlays';
+  tab.style.position = 'relative';
+
+  const {tabLabel, tabPane} = W.userscripts.registerSidebarTab("imageoverlays");
+
+  tabLabel.innerHTML = '<span class="fa fa-picture-o"></span>';
+  tabLabel.title = I18n.t('image_overlays.tab_title');
+  tabPane.innerHTML = tab.outerHTML;
```

Comment:
tabPane is already a DOM div element. Only needs to have innerHTML added. However, we want all of our HTML from {tab} put in, so we use tab.outerHTML.